### DISCUSSION TITLE
Which endpoint to use for various Kratos APIs?

### DISCUSSION THREAD

**Discussion by Cavaler** [ory/kratos#4108] [Q&A]
Is there any documentation, which Kratos APIs are accessible on which endpoint (admin, public, or maybe both)?
I see not a single word about that in [official API reference](https://www.ory.sh/docs/kratos/reference/api).

Maybe at least there's documentation like "/serf-service is public but /admin is (obviously) admin"? Or everything not with /admin is public?

**Comment by vinckr** [comment]
Hello @Cavaler 

identity is the admin API and frontend is the self-service / public API.

When you use Ory Network you need an access token for the identity API - if you self-host Ory you need to implement some access control for the admin APIs yourself, out of the box everything is exposed.
